explorer/fuzzing/ast_to_proto.cpp

```diff
@@ -110,11 +110,9 @@ static auto ExpressionToProto(const Expression& expression)
     -> Fuzzing::Expression {
   Fuzzing::Expression expression_proto;
   switch (expression.kind()) {
-    case ExpressionKind::InstantiateImpl:
-    case ExpressionKind::ValueLiteral: {
-      // These do not correspond to source syntax.
+    case ExpressionKind::ValueLiteral:
```

Comment:
Please re-add braces, they're a style choice (for consistency with Carbon):

"Always use braces for conditional, switch, and loop statements, even when the body is a single statement."

https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/cpp_style_guide.md#syntax-and-formatting

Reply:
Will do. FWIW, we have substantial deviation from this rule already; filed #2251 for that.